kobo/apps/long_running_migrations/jobs/0004_back_fill_asset_search_field_for_owner_label.py

```diff
@@ -0,0 +1,42 @@
+# Generated on 2024-12-19 15:34
+from django.db.models import Prefetch
+from more_itertools import chunked
+
+from kobo.apps.organizations.models import Organization
+from kpi.models.asset import Asset
+
+
+def run():
```

Comment:
Should we have unit tests for at least some of these migrations? Though, I would expect them to get tested when devs pull from main and the local environment executes them from time to time. But maybe just for QA purposes, either having unit tests or reproducible previews.

Reply:
We decided to do simple tests (e.g.: validate the data is correctly added) in another PR.